recipe/bld.bat

```diff
@@ -1,6 +1,11 @@
+set "CMAKE_GENERATOR=NMake Makefiles"
```

Comment:
I found this needed to be set manually. Is this expected @isuruf?

When not setting this value, it appears to be [overriden with a VS generator]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=88812&view=logs&jobId=5be07ae1-d8ba-5406-47b6-8e3a3a12f825&taskId=0d1d4714-3221-52cd-cfdb-f62a891e036b&lineStart=408&lineEnd=409&colStart=1&colEnd=1 ), which can also be seen in this [CI build log]( https://github.com/conda-forge/pyflann-feedstock/files/3802432/win_python37_use_variant_cmake_generator.txt ). This causes the build to fail.

Reply:
Yes, vs2015_win-64 overrides this even though this value is in conda_build_config.yaml